frontend/src/components/shared/layout/Layout.tsx

```diff
@@ -6,14 +6,23 @@ import './Layout.scss';
 interface IProps {
   pageTitle: string;
   children: ReactNode;
+  countFriends?: number;
 }
 
-const Layout: React.FC<IProps> = ({ pageTitle, children }: IProps) => {
+const Layout: React.FC<IProps> = ({ pageTitle, children, countFriends }: IProps) => {
+  const pageName = Boolean(countFriends) ? (
+    <>
+      <h1 className="page-title-friends">{pageTitle}</h1>
+      <span>({countFriends})</span>
+    </>
+  ) : (
+    <h1 className="page-title">{pageTitle}</h1>
+  );
```

Comment:
can we display inline block by default? then no need in trinary operator here
or

`<>
      <h1 className='class condition'>{pageTitle}</h1>
      {Boolean(countFriends) && (<span>({countFriends})</span>)}
    </>`